mjpc/tasks/shadow_reorient/task.xml

```diff
@@ -47,7 +47,8 @@
     <framepos name="trace0" objtype="body" objname="cube"/>
   </sensor>
 
-  <include file="../common_assets/reorientation_cube.xml"/>
+  <!-- from: common_assests -->
+  <include file="cube_modified.xml"/>
```

Comment:
now the cube xml lives in the task folder, but its assets are back in the cube folder; I think it's a bad idea.  Can you putput the modified file in the same cube dir?

Reply:
Let me know if the latest change is what you are thinking.